lcls-twincat-optics/lcls_twincat_optics_plc/POUs/Helpers/FB_MirrorTwoCoatingProtection.TcPOU

```diff
@@ -104,19 +113,45 @@ END_IF
 IF nCurrentEncoderCount <= nLowerCoatingBoundary THEN
     ffLowerCoating.i_xOK := (neVRange AND nLowerCoatingBitMask) = neVRange;
     ffUpperCoating.i_xOK  := TRUE;
+    IF bMirrorTempFaults THEN
+        ffLowerCoatingLTemp(fFaultThreshold:=aDbStateParams[1].stReactiveParams.nTempSP,
```

Comment:
minor nitpick: in general I'd advocate for using a local enum instead of magic numbers, but with only 1 and 0 this is totally manageable in this FB

ref https://alltwincat.com/2021/11/16/local-enumerations/ for knowledge spreading

Reply:
Good idea. I think it's worth updating that before merging. Will update that today.

Reply:
Updated this. This is the final version. Do you mind taking another look? @ZLLentz